shared/android/app/src/main/java/io/keybase/ossifrage/MainApplication.java

```diff
@@ -72,13 +117,51 @@ public void onCreate() {
         }
     }
 
+   /**
+   * Waits for Loading to complete, also called a Time-To-Interaction (TTI) event. To indicate TTI
+   * completion, add a prop nativeID="tti_complete" to the component whose appearance indicates that
+   * the initial TTI or loading is complete
+   */
+  @RequiresApi(api = Build.VERSION_CODES.Q)
+  private void addTTIEndListener() {
+    Log.d("Marker", "Setting listener");
+    Trace.beginAsyncSection("Starting App", 0);
+    ReactFindViewUtil.addViewListener(
+        new ReactFindViewUtil.OnViewFoundListener() {
+          @Override
+          public String getNativeId() {
+            // This is the value of the nativeID property
+            return "tti_complete";
+          }
+
+          @Override
+          public void onViewFound(final View view) {
+            Log.d("Marker", "Found view");
+            // Once we find the view, we also need to wait for it to be drawn
+            view.getViewTreeObserver()
+                // TODO (axe) Should be OnDrawListener instead of this
+                .addOnPreDrawListener(
+                    new ViewTreeObserver.OnPreDrawListener() {
+                      @Override
+                      public boolean onPreDraw() {
+                        view.getViewTreeObserver().removeOnPreDrawListener(this);
+                        Trace.endAsyncSection("Starting App", 0);
```

Comment:
We ran into cases where during experiments, people may miss this element. Should this have a timeout ?